tools/codecome.py

```
@@ -345,6 +345,110 @@ def count_findings() -> Dict[str, int]:
+
+
+def check_codeql_status() -> int:
+    """Check CodeQL configuration and last recorded artifact state."""
+    print()
+    print(C.header("CodeQL:"))
+
+    try:
+        from codeql.config import resolve_config
+        from codeql.artifacts import check_artifacts
+        from codeql.packs import load_codeql_plan
+    except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)